### PR TITLE
Add build process

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Fixes https://github.com/giantswarm/prometheus-kemp-exporter/issues/3

### CODE REVIEW

**Comment by JosephSalisbury** [comment]
Gotta love that travis hustle

**Comment by JosephSalisbury** [comment]
@xh3b4sd RFR plz

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        - ./builder-cache
+        
+notifications:
+    email: false
+
+before_install:
+    # Grab builder from the cache, or download it if we can't.
+    - cp ./builder-cache/builder-$BUILDER_VERSION ./builder || wget https://downloads.giantswarm.io/builder/$BUILDER_VERSION/builder && chmod +x ./builder
+    - chmod +x ./builder
+    - export PATH=$PATH:$PWD
```
Comment (xh3b4sd) [inline .travis.yml]: What about having this added to the Makefile? Then the CI target can depend on such setup targets.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,5 @@
 prometheus-kemp-exporter
+.gobuild/
+bin-dist/
+build/
```
Comment (xh3b4sd) [inline .gitignore]: For what do we need `bin-dist` and `build`?

File: README.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # prometheus-kemp-exporter
 
+[![Build Status](https://api.travis-ci.org/giantswarm/prometheus-kemp-exporter.svg)](https://travis-ci.org/giantswarm/prometheus-kemp-exporter)
```
Comment (xh3b4sd) [inline README.md]: What about the other badges we have in the other open source repos?

**Comment by xh3b4sd** [comment]
Minor nitpicking. LGTM

File: .gitignore (hunk 2 above)
Comment (JosephSalisbury) [inline .gitignore]: Any files in `bin-dist/` are uploaded to the GitHub release. I've been using `build/` as a temporary directory for the uncompressed tarballs, so they don't clutter `bin-dist/`.

File: README.md (hunk 3 above)
Comment (JosephSalisbury) [inline README.md]: They're getting there :D

File: .travis.yml (hunk 1 above)
Comment (JosephSalisbury) [inline .travis.yml]: I don't like the idea of downloading stuff in Makefiles, that feels janky / surprising.

**Pull request merged**